docs/Frequently Asked Questions/Can I Integrate SDL into a React Native App/index.md

```diff
@@ -0,0 +1,231 @@
+
+!!! NOTE
+You must make sure you have [Native Modules](https://facebook.github.io/react-native/docs/native-modules-setup) installed as a dependency in order to use 3rd party APIs in a React Native application. If this is not done your app will not work with SmartDeviceLink.
+!!!
+
+!!! NOTE
+Native API methods are not exposed to Javascript. Follow the guides for more in depth explanations on how to export the methods you wish to expose to your application.
+!!!
+
+!!! NOTE
+Make sure you have followed the [Getting Started](https://smartdevicelink.com/en/guides/iOS/getting-started/installation/) and have `SmartDeviceLink` installed on the native side. 
+!!!
+
+## Integerating Basics
+Native API methods are not exposed automatically to Javascript. This means you must expose methods you wish to use to use own your own.  To get started you must implement `RCTBridgeModule` protocol. Please follow [Integrating Basics](https://smartdevicelink.com/en/guides/iOS/getting-started/integration-basics/) for the basic setup. This is the necessary starting point in order to contunie with this example. Please make sure you also set up a simple UI with buttons and some textfields.
+
+## Creating the RCTBridge
+To create a native module you must implement the `RCTBridgeModule` protocol. Update ProxyManager to include `RCTBridgeModule`.
+
+!!! NOTE
+Swift will have a few more steps in order to achieve this since swift does not support macros.
+!!!
+
+##### Objective-C
+###### ProxyManager.h
+```objc
+#import <React/RCTBridgeModule.h>
+@interface ProxyManager : NSObject <RCTBridgeModule>
+<#...#>
+@end
+```
+###### ProxyManager.m
+A  `RCT_EXPORT_MODULE()` macro must be added to the implementation file to expose the class to React Native.
+```objc
+@implementation ProxyManager
+RCT_EXPORT_MODULE();
+<#...#>
+@end
+```
+##### Swift
+First you must add  `#import "React/RCTBridgeModule.h"` to your `Bridging Header` before you move forward. When creating a swift appication xcode ask if it should create a header file for you. You must include this bridging header for your React Native app to work. You can create this file manually if you choose to do so. 
+
+```swift
+@objc(ProxyManager)
+class ProxyManager: NSObject {
+<#...#>
+}
+```
+To expose the Swift class to React Native you must create an Objective-C file in order to use React Native macros.
+
+```objc
+// ProxyManager.m
+#import "React/RCTBridgeModule.h"
+
+@interface RCT_EXTERN_MODULE(ProxyManager, NSObject)
+@end
+```
+## Exposing Methods
+Since it is recommended that the `ProxyManager` is used for inital set up only we suggest creating a new class that exposes your methods and post a notification from the `ProxyManager` class.
```

Comment:
I found it to make creating the app much easier since we recommended they use the App Delegate to start the singleton. If they also create an instance of ProxyManger in the javascript side they would have to constantly dependency inject the reference in order to use the `ProxyManager`. Keeping the `ProxyManger` on the native side only makes managing it much easier.